lib/modules/manager/mint/readme.md

```diff
@@ -0,0 +1,14 @@
+Renovate supports updating Mintfiles.
+
+[`yonaskolb/Mint` repository on GitHub](https://github.com/yonaskolb/Mint)
```

Comment:
```suggestion
Go to the [`yonaskolb/Mint` repository on GitHub](https://github.com/yonaskolb/Mint) to learn more about the Mint package manager.
```

I think this is clearer about what you'll see when you click the link, and why you may want to go to the official repository. 😄